lib/flatfile.ts

```diff
@@ -45,13 +45,15 @@ const flatfileClient = (workflow: WorkflowType) => {
 
   switch (workflow) {
     case WorkflowType.Onboarding:
-      token = process.env.ONBOARDING_FLATFILE_API_KEY;
+      token =
+        process.env.FLATFILE_API_KEY || process.env.ONBOARDING_FLATFILE_API_KEY;
       break;
     case WorkflowType.FileFeed:
-      token = process.env.FILEFEED_API_KEY;
+      token = process.env.FLATFILE_API_KEY || process.env.FILEFEED_API_KEY;
       break;
     case WorkflowType.Embedded:
-      token = process.env.EMBEDDED_FLATFILE_API_KEY;
+      token =
+        process.env.FLATFILE_API_KEY || process.env.EMBEDDED_FLATFILE_API_KEY;
```

Comment:
Once we have these set in heroku we can just reference them directly, don't need the ||

Reply:
ok